upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -202,16 +202,13 @@ func NewCluster(opt *NewClusterOptions, clientset simple.Clientset) (*NewCluster
 		},
 	}
 
-	if channel.Spec.Cluster != nil {
-		cluster.Spec = *channel.Spec.Cluster
```

Comment:
I think the intention here was to be able to populate ClusterSpec from channels.

Reply:
Except it then overwrites large sections of the config.

The current code works only because it overwrites the kubenet setting in the channel with a blank networking struct.